src/amo/components/AddonCompatibilityError/index.js

```diff
@@ -37,14 +36,19 @@ export class AddonCompatibilityErrorBase extends React.Component {
   render() {
     const {
       i18n,
-      lang,
       log,
       minVersion,
       reason,
       userAgentInfo,
     } = this.props;
-    const downloadUrl = `https://www.mozilla.org/${lang}/firefox/`;
-    let message;
+
+    const queryString = [
```

Comment:
I'm pretty sure we have an actual `makeQueryParam` helper or something which *might* be nicer just because it already has tests and takes an object as an argument, but this is fine 😄

Reply:
ah, indeed `makeQueryString` it is called 😓  I'll use it!